src/components/nav-sidebar/NavList.js

```diff
@@ -2,25 +2,47 @@
 import * as React from 'react';
```

Comment:
I would also change caret to match https://github.com/box/box-ui-elements/pull/1218

Reply:
looks like recent designs still specify that left side bar carets expand and collapse similar to file explorer conventions like in Finder.app and Windows Explorer. Probably good to double-check if this difference is intended

<img width="134" alt="Screen Shot 2019-05-31 at 11 33 54 AM" src="https://user-images.githubusercontent.com/3056447/58726983-fbd74400-8397-11e9-922e-5135b9ebcc74.png">

Reply:
The native design is also that way, look at the `<details>` html5 tag.

Reply:
That's a good point. I'm going to stick to what is in the design which is what @booc0mtaco pointed out (similar to Mac finder).